browser/brave_browser_process_impl.h

```diff
@@ -113,6 +113,7 @@ class BraveBrowserProcessImpl : public BraveBrowserProcess,
   brave_shields::AdBlockService* ad_block_service() override;
   https_upgrade_exceptions::HttpsUpgradeExceptionsService*
   https_upgrade_exceptions_service() override;
+  void webcompat_exceptions_service() override;
```

Comment:
this is not a getter, i think we just don't need this function

Reply:
removed